src/i18n/locale/en.json

```diff
@@ -935,14 +935,17 @@
   "components.Settings.SettingsMain.validationApplicationTitle": "You must provide an application title",
   "components.Settings.SettingsMain.validationApplicationUrl": "You must provide a valid URL",
   "components.Settings.SettingsMain.validationApplicationUrlTrailingSlash": "URL must not end in a trailing slash",
+  "components.Settings.SettingsNetwork.advancedNetworkSettings": "Advanced Network Settings",
   "components.Settings.SettingsNetwork.csrfProtection": "Enable CSRF Protection",
   "components.Settings.SettingsNetwork.csrfProtectionHoverTip": "Do NOT enable this setting unless you understand what you are doing!",
   "components.Settings.SettingsNetwork.csrfProtectionTip": "Set external API access to read-only (requires HTTPS)",
   "components.Settings.SettingsNetwork.dnsServers": "Custom DNS Servers",
   "components.Settings.SettingsNetwork.dnsServersTip": "Comma-separated list of custom DNS servers, e.g. \"1.1.1.1,[2606:4700:4700::1111]\"",
+  "components.Settings.SettingsNetwork.docs": "documentation",
   "components.Settings.SettingsNetwork.forceIpv4First": "IPv4 Resolution First",
   "components.Settings.SettingsNetwork.forceIpv4FirstTip": "Force Jellyseerr to resolve IPv4 addresses first instead of IPv6",
   "components.Settings.SettingsNetwork.network": "Network",
+  "components.Settings.SettingsNetwork.networkDisclaimer": "Network parameters from your container/system should be used instead of this setting. See the {docs} for more information.",
```

Comment:
```suggestion
  "components.Settings.SettingsNetwork.networkDisclaimer": "Network parameters from your container/system should be used instead of these settings. See the {docs} for more information.",
```